.mailmap

```diff
@@ -8,22 +8,32 @@
 #
 #
 Aaron Bassett <abassett@gmail.com>
-Aashish Sharma <aasharma@redhat.com>
+Aaryan Porwal <aaryanporwal2233@gmail.com> aaryanporwal <NOT@FOUND>
+Aashish Sharma <aasharma@redhat.com> <66050535+aaSharma14@users.noreply.github.com>
+Aashish Sharma <aasharma@redhat.com> <aasharma@li-e74156cc-2f67-11b2-a85c-e98659a63c5c.ibm.com>
+Aashish Sharma <aasharma@redhat.com> <aashishsharma@fedora.redhat.com>
+Aashish Sharma <aasharma@redhat.com> <aashishsharma@localhost.localdomain>
+Abhishek Lekshmanan <abhishek.lekshmanan@cern.ch> <abhishek.l@cern.ch>
 Abhishek Lekshmanan <abhishek@suse.com> <abhishek.lekshmanan@gmail.com>
 Abhishek Lekshmanan <abhishek@suse.com> <alekshmanan@suse.com>
 Adam C. Emerson <aemerson@redhat.com>
 Adam King <adking@redhat.com> <kingamk3@gmail.com>
+Adam King <adking@redhat.com> <adking@redhat,com>
```

Comment:
the second email it just a typo and the line above this correctly has my redhat and personal email, so you can probably drop the commit updating this for me.

Reply:
thanks @adk3798. there are commits or reviews that used this misspelled address, so i added it here to make sure they still get credited under your correct name